docs/_docs/04_reference/05-strict-mode.md

```diff
@@ -57,40 +57,40 @@ flag to a value that's specific to a particular strict control.
 This can allow you to gradually increase your confidence in the future compatibility of your Terragrunt usage.
 
 ```bash
-$ terragrunt plan-all --strict-control apply-all
```

Comment:
We can't do this, @levkohimins . All strict controls need to be specific and complete-able.

`deprecated-commands` is not something that has a lifecycle.

Very specific controls like `plan-all` do, and so do more general strict controls like `cli-redesign` (as we'll eventually no longer support the `plan-all` command, and we'll eventually complete the RFC for the CLI Redesign).

Users have no context for the timeline on `deprecated-commands`.